src/utils.cpp

```diff
@@ -240,18 +255,25 @@ int light_level(struct room_data *room)
       break;
   }
 
+  // Indoor rooms have no weather or time-based light influences.
   if (room->sector_type == SPIRIT_HEARTH)
```

Comment:
Why are we using SPIRIT_HEARTH to check for indoor rooms instead of `ROOM_FLAGGED(room, ROOM_INDOORS)`?